README.md

```diff
@@ -129,6 +129,26 @@ stackit config list
 
 You can also edit the configuration file manually.
 
+## Customization
+
+### Using **less** as a pager
+
+STACKIT CLI uses the `less` as default pager. 
+When using `less` as a pager, STACKIT CLI will automatically pass following options
+- -F, --quit-if-one-screen - Less will automatically exit if the entire file can be displayed on the first screen.
+- -S, --chop-long-lines - Lines longer than the screen width will be chopped rather than being folded.
+- -w, --hilite-unread - Temporarily highlights the first "new" line after a forward movement of a full page.

```

Comment:
It's missing the newly introduced argument "-R" for the colors

Reply:
Added a line to explain the -R option.